lightbug_http/header.mojo

```diff
@@ -389,6 +389,14 @@ struct RequestHeader:
                 return n
 
 
+struct StatusCode:
+    alias OK = 200
+    alias MOVED_PERMANENTLY = 301
+    alias FOUND = 302
+    alias TEMPORARY_REDIRECT = 307
+    alias PERMANENT_REDIRECT = 308
+    alias NOT_FOUND = 404
```

Comment:
Potentially we could also handle 303 (See Other), 300 (Multiple Choices) and 304 (Not modified), although the logic for latter two might be more complicated. But can also leave it for now and implement in the future

https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections